src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/client/WorkforceAllocationsToDeliusApiClient.kt

```diff
@@ -152,9 +152,14 @@ data class DeliusCaseDetail(
 data class Event(val number: String)
 
 data class ProbationStatus(val description: String)
-data class InitialAppointment(val date: LocalDate?)
+data class InitialAppointment(val date: LocalDate?, val staff: Staff?)
+
+data class Staff @JsonCreator constructor(
+  val name: Name?,
```

Comment:
I can understand why `start` might be `null` and why that has been made nullable therefore but suspect that if there is a value for staff they will always have a name. Maybe we want to double check this with them but do not think we should be overly cautious or will end up with nullable everything.

Happy to chat on this though :)